index.js

```diff
@@ -1616,6 +1616,29 @@
     return R.map(get(type, key), xs);
   });
 
+  //# unfoldr :: (b -> Maybe (a, b)) -> b -> [a]
+  //.
+  //. The unfoldr function is a `dual' to reduce: while reduce "reduces" a list
+  //. to a summary value, unfoldr builds a list from a seed value. The function
+  //. takes the element and returns an instance of Nothing if it is done
+  //. producing the list or returns an instance of Just [a, b], in which case,
+  //. a is a prepended to the list and b is used as the next element in a
+  //. recursive call.
```

Comment:
It took well over an hour across several sittings during a two-week period, but I've succeeded in writing a description of this complex function in the style used elsewhere:

```javascript
  //. Takes a function and a seed value, and returns a list generated by
  //. applying the function repeatedly. The list is initially empty. The
  //. function is initially applied to the seed value. Each application
  //. of the function should result in either:
  //.
  //.   - a Nothing, in which case the list is returned; or
  //.
  //.   - Just a pair, in which case the first element is appended to
  //.     the list and the function is applied to the second element.
```